frontend/lib/api-bridge-client.js

```diff
@@ -2,8 +2,11 @@
 
 import 'fetch'
 import * as Constants from 'lib/constants'
+import APIWrapper from '@dadi/api-wrapper-core'
+import Cookie from 'js-cookie'
 
-const APIWrapper = require('@dadi/api-wrapper-core')
+// The cookie name to store the bearer token
+const KEY_BEARER_TOKEN = 'bearer_token'
```

Comment:
Perhaps I'm missing something, but this seems to be suggesting that the bearer token is to be made available client-side. Isn't that a security vulnerability?

Reply:
That's what I was told to do - either as a cookie or localstorage - but was asked to make it a cookie - how would it not be, if we're changing to this bearer token method of hitting from client-side?

Reply:
The ACL piece introduced in API 4.0 allows clients (and their bearer tokens) to have a limited set of permissions, so I don't see this as a security vulnerability. Sure, they can intercept the bearer token being used by peeking into the Ajax requests, but it's _their_ token, which will not give them access to anything they can't already do via the Publish interface.